platform/harmony/stake.go

```diff
@@ -0,0 +1,94 @@
+package harmony
+
+import (
+	"github.com/trustwallet/blockatlas/pkg/blockatlas"
+	"github.com/trustwallet/blockatlas/pkg/errors"
+	"github.com/trustwallet/blockatlas/pkg/logger"
+	services "github.com/trustwallet/blockatlas/services/assets"
+)
+
+func (p *Platform) GetValidators() (blockatlas.ValidatorPage, error) {
+	results := make(blockatlas.ValidatorPage, 0)
+	validators, err := p.client.GetValidators()
+	if err != nil {
+		return results, err
+	}
+
+	apr, err := p.client.GetAPR()
+	if err != nil {
+		apr = Annual
+	}
+
+	for _, v := range validators.Validators {
+		if val, ok := normalizeValidator(v, apr); ok {
+			results = append(results, val)
+		}
+	}
+	return results, nil
+}
+
+func (p *Platform) GetDetails() blockatlas.StakingDetails {
+	apr, err := p.client.GetAPR()
+	if err != nil {
+		apr = Annual
+	}
+	return getDetails(apr)
+}
+
+func (p *Platform) GetDelegations(address string) (blockatlas.DelegationsPage, error) {
+	results := make(blockatlas.DelegationsPage, 0)
+	delegations, err := p.client.GetDelegations(address)
+	if err != nil {
+		return nil, err
+	}
+
+	validators, err := services.GetValidatorsMap(p)
+	if err != nil {
+		return nil, err
+	}
+	results = append(results, NormalizeDelegations(delegations.List, validators)...)
```

Comment:
why did you create an empty result slice only to append the normalized delegations?

Reply:
my bad. removed.